angular/_blueTheme.scss

```diff
@@ -109,6 +109,20 @@
             background-color: map-get($pxb-white, 200);
         }
     }
+    .mat-form-field .mat-form-field-label {
+        color: map-get($pxb-black, 500);
+    }
+    .mat-form-field .mat-hint {
+        color: map-get($pxb-gray, 500);
+    }
+
+    /* Mat Form Field hover ripple */
+    .mat-form-field-ripple {
+        background-color: map-get($pxb-gray, 500);
+    }
+    .mat-form-field-appearance-outline .mat-form-field-outline-thick {
+        color: map-get($pxb-gray, 500);
+    }
```

Comment:
Instead of hardcoding these from the color palette package, can we use variables from the theme foreground itself (so we only need to update these in one place for future changes):

![image](https://user-images.githubusercontent.com/29152776/103281924-37862580-49a2-11eb-829a-fa8b72274d2c.png)

Reply:
I've updated this PR to use colors from the palette where we can.